src/styles/utils/fonts.scss

```diff
@@ -0,0 +1,42 @@
+@font-face {
```

Comment:
this file should have name starting with `_` as all of the utils -> `_fonts.scss`